### ISSUE TITLE
``logout.backchannel.path`` fails when ``http.root-path`` is present - Again

### ISSUE THREAD

**Issue by AB-xdev** [quarkusio/quarkus#42990]
### Describe the bug

Basically the same problem as in https://github.com/quarkusio/quarkus/issues/42483 happens again with the latest release (3.14.1).
The fix was likely incomplete, as we also encountered the problem in 3.13.3.

### How to Reproduce?

We created a full PoC repo with detailed information, how the bug can be reproduced:
https://github.com/AB-xdev/quarkus-backchannel-logout-poc

### Quarkus version or git rev

3.14.1

**Comment by geoand** [comment]
cc @sberyozkin

**Comment by sberyozkin** [comment]
Thanks @AB-xdev for preparing a POC repo.

**Comment by AB-xdev** [comment]
Problem is likely caused by 
https://github.com/quarkusio/quarkus/blob/15a7158313c6d336bb457b5c9807fbd7a5e95f08/extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/BackChannelLogoutHandler.java#L49

While debugging ``RouterImpl#route`` I noticed that nearly all paths are always registered without the ``/backend``-rootpath-prefix.
Only the backchannel-logout endpoints are inconsistent in this behavior and get registered **with** it.<br/>
When I use the debugger to replace ``/backend/oidc/wim/back-channel-logout`` with ``/oidc/wim/back-channel-logout`` during  registration and then perform a backchannel logout via keycloak everything works as expected:
```
2024-09-03 17:03:22,329 DEBUG [io.qua.oid.run.BackChannelLogoutHandler] (vert.x-eventloop-thread-4) Back channel logout request for the tenant wim received
// No further errors
// Browser gets redirect to login during next request
```

**Comment by sberyozkin** [comment]
@AB-xdev Do you have some concrete examples in the Quarkus code where the root path is not used at the router registration but which can be successfully accessed with the http root path configured ?

**Comment by cescoffier** [comment]
it should be automatically mounted at the right path - except if the route is an absolute route.

**Comment by sberyozkin** [comment]
Thanks @cescoffier, so we'll just need to remove that HTTP root prefix.
@AB-xdev Hi, since you are are already looking at that code, can you give me a favor, and remove `getRootPath()` at the route setup and confirm with your POC it fixes it ? PR will be welcomed too, thanks

**Comment by sberyozkin** [comment]
@AB-xdev I'll add a proper test as well to support your PR should you with it, thanks

**Comment by AB-xdev** [comment]
I created a PR here that removes the 16 characters: https://github.com/quarkusio/quarkus/pull/43000

Note however that setting it up took me longer than doing the actual change so for future small changes I would appreciate it if you could do them instead :)

**Comment by AB-xdev** [comment]
Rechecked it and can confirm that this is fixed indeed now fixed in 1.14.3.

Thank you for the very quick response and release 👍

**Comment by geoand** [comment]
Great, thanks for checking!
